docker/linksets/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.4'
+
+services:
+  linksets:
+    build: 
+      context: linksets-builder 
+      target: python-specific
+    env_file:
+      - ./.env
```

Comment:
The `.env` file you are pulling here is a tini bit confusing. The `.env` file next to a docker-compose file is automatically applied to docker-compose for use inside the docker-compose file construction. By explicitly calling it out here, you are subverting the normal use of the `.env` file and forcing the use of those variables. To clear things up, I might suggest renaming the file to `defaults.env` or something, freeing up `.env` for its normal use.

Reply:
kept it as .env as I'm not sure what other purpose a .env file might have but using it in combination with the common .env